content/docs/reference/api/aut/_index.md

```diff
@@ -1362,6 +1433,7 @@ curl --location --request GET 'https://rpc1.bakerloo.autonity.org/' \
 '0x5fe87ee4f61da6e640aec02ce818cdcd30b8cb13'
 {{< /tab >}}
 -->
+
 ##  getUnbondingReq
```

Comment:
This is depreciated, the new way of tracking bonding and unbonding is through events

Reply:
Oh actually I think it's because the improved unbonding hasn't been documented yet

Reply:
Yes. I am making edits for that in the `improved-unbonding` branch and PR!

i.e. for each feature addition or update, a discrete PR.

Reply:
got it